vision/Blobdetection/simpleDetection.py

```diff
@@ -0,0 +1,70 @@
+
```

Comment:
Lets put this code in vision/camera! I can have someone format all of the types of camera streams so they are generic interfaces for all the vision algorithms. Rename it to webcam.py or something.